wfe/web-front-end.go

```diff
@@ -207,13 +211,20 @@ func (wfe *WebFrontEndImpl) NewCertificate(response http.ResponseWriter, request
 
 	var init core.CertificateRequest
 	if err = json.Unmarshal(body, &init); err != nil {
+		fmt.Println(err)
```

Comment:
Good news! This is an auditable event (Receipt of improper messages) See Issue #62.